frontend/Common/RunUtils.js

```diff
@@ -5,3 +5,7 @@ export const getScyllaPackage = function (packages) {
 export const getKernelPackage = function (packages) {
     return packages.find((pkg) => pkg.name == "kernel");
 };
+
+export const getUpgradedScyllaPackage = function(packages) {
+    return packages.find((pkg) => pkg.name == "scylla-server-upgraded");
```

Comment:
I think upgraded version is stored in `scylla-server` package. Upgraded version keeps version after upgrade.

Reply:
Doesn't seem to be the case, I'm looking at the latest rolling upgrade longevity and this is what I see:
![image](https://user-images.githubusercontent.com/7761415/180735810-84d6daa2-1436-4b7c-bb4b-230343a18d47.png)

Reply:
That's what I mean, 5.0.1 is version after upgrade - so shouldn't 'Upgraded scylla version' should take value from 'scylla-server` which points to start version (5.0 in this case)?

Reply:
The version in `scylla-server` is currently being output as `Base scylla version`, as for phrasing of this one, what would you suggest to clear it up?

Reply:
> That's what I mean, 5.1.dev is version after upgrade - so shouldn't 'Upgraded scylla version' should take value from 'scylla-server` which points to start version (5.0 in this case)?

we should be upgrading (if the run was in the latest master one) from `5.0.0` (latest released OSS version) to latest master one (`5.1.dev`)

Reply:
ok, just wanted to confirm that :sweat_smile: